node/network/collator-protocol/src/collator_side.rs

```diff
@@ -16,15 +16,16 @@
 
 use std::collections::HashMap;
 
-use super::{TARGET,  Result};
+use super::{LOG_TARGET,  Result};
 
-use futures::{StreamExt, channel::oneshot, task::Poll};
+use futures::{StreamExt, task::Poll};
 use log::warn;
 
 use polkadot_primitives::v1::{
 	CollatorId, CoreIndex, CoreState, Hash, Id as ParaId, CandidateReceipt,
 	PoV, ValidatorId,
 };
+#[allow(unused_imports)]
```

Comment:
Why do we need this?

Reply:
The warning was introduced in ed77d42:
```
warning: unused imports: `RuntimeApiMessage`, `RuntimeApiRequest`
  --> node/network/collator-protocol/src/collator_side.rs:31:41
   |
31 |         AllMessages, CollatorProtocolMessage, RuntimeApiMessage, RuntimeApiRequest,
   |                                               ^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```
Likely, this is related to macro requiring these types to be present. Removing the imports leads to an error.
What is the proper fix here? Somehow fix the macro I guess?

Reply:
I checked it and the errors are coming from the tests. You need to import the messages in the tests module.

Reply:
Thanks!